registry/server/index.ts

```diff
@@ -1,16 +1,39 @@
 import path from 'path';
 
+['log', 'warn', 'error'].forEach(function (method) {
```

Comment:
this code looks a little bit not transparent to me. Could you please leave a comment why is it written in this way and explaining what does it do